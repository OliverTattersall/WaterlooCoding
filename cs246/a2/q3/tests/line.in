a 0 b 4 2 1 4
pc0
a 0 b 4 2 6 0
pc0
