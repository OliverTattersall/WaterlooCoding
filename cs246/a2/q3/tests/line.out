Dimensions: 3x8
Number of rectangles: 1
	[Blue (4,2) 1x4]
Dimensions: 8x8
Number of rectangles: 2
	[Blue (4,2) 1x4]
	[Blue (4,2) 6x0]
